Api/Src/Services/AuthService.ts

```diff
@@ -0,0 +1,70 @@
+import {JWT_TOKEN_EXPIRESIN} from "../../Constants";
+import {UserModel} from "../Models/User";
+import { UserType} from "../Models/User";
+import * as mongoose from 'mongoose';
+import UserRepository from "../Repositories/User";
+import * as bcrypt from "bcrypt";
+const jwt = require('jsonwebtoken');
+
+type User = UserModel & mongoose.Document;
+
+export default class AuthService {
+    private saltRounds: number = 10;
```

Comment:
This was already defined in Constants.ts as SALT_ROUNDS.

Reply:
Actually I deleted it from constants - maybe it's better to avoid external dependecies inside the class? I think - that kind of field can be placed inside the class.  ?

Reply:
It is part of the implementation so you're right, it can stay here.